spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/RabbitUtils.java

```diff
@@ -288,4 +290,29 @@ public static boolean isExchangeDeclarationFailure(Exception e) {
 		}
 	}
 
+	/**
+	 * Determine whether a message should be requeued; returns true if ex is a
```

Comment:
If `exception`.
I don't think that it is a good practice to abbreviate in public OpenSource Docs.